packages/ckeditor5-font-mapper/README.md

```diff
@@ -0,0 +1,133 @@
+# CKEditor Plugin: Font Mapper
+
+The Font Mapper plugin extends the
+[CKEditor 5 Clipboard Feature][cke5:docs:clipboard:feature].
+
+When pasting rich text from external sources into the editor, some characters of
+the pasted text might originate from an unsupported font. This plugin maps such
+characters to their named entities or Unicode equivalents using a mapping table.
+
+By default, this plugin already contains a mapping table for the Word `Symbol` font, and automatically
+converts input content (for instance while pasting from Microsoft Word) accordingly.
+This mapping table can be replaced or extended.
+
+
+## Clipboard Pipeline Integration
+
+The Font Mapper plugin listens to the ClipboardPipeline's `inputTransformation` event with `normal` priority. This way, it works fine in combination with CKEditor's `PasteFromOffice` plugin, which also listens to the same event, but has a higher priority.
+
+Both plugins manipulate the event's `data.content` object. This is why it is important to register the event listener with lower priority. Otherwise the `PasteFromOffice` plugin would just override the data changed in this plugin.
+
+
+## Example: Copy & Paste from Microsoft Word
+
+This plugin was originally created to prevent broken characters to appear when pasting content from a Word document. In this case, characters written in the `Symbol` font would not be converted correctly during the transfer.
```

Comment:
We may want to either explicitly state _Microsoft Word_ (but see, if there is any TM to add) or just use the generic term "office application".